File: 00-user_defined_data_type/user_defined_data_type.py
# -*- coding: utf-8 -*-

class Client:
    def __init__(self, name:str, id:str, adress:str, credit:float):
        self.name = name
        self.id = id
        self.adress = credit
        self.credit = credit

client_1 = Client('Anthony Luzquiños',
                    '0000000001',
                    'Calle 1, Distrito 1, Ciudad 1, País 1',
                    1000000)

print(f'The client\'s name is: {client_1.name}.\n'+
        f'The client\'s id is: {client_1.id}.\n'+
        f'The client\'s adress is: {client_1.adress}.\n'+
        f'The client\'s credit is: {client_1.credit}.')

File: 00-user_defined_data_type/user_defined_data_type.js
'use strict'

class Client{
    constructor(name, id, adress, credit){
        this.name = name
        this.id = id
        this.adress = adress
        this.credit = credit
    }
}

let client_1 = new Client(
                    'Anthony Luzquiños',
                    '0000000001',
                    'Calle 1, Distrito 1, Ciudad 1, País 1',
                    1000000)

console.log('The client\'s name is: '+client_1.name+'.\n'
            +'The client\'s id is: '+client_1.id+'.\n'
            +'The client\'s adress is: '+client_1.adress+'.\n'
            +'The client\'s credit is: '+client_1.credit+'.')
